variables.tf

```diff
@@ -122,6 +122,12 @@ variable "iap" {
   default     = null
 }
 
+variable "ssl_policy" {
+  type        = string
+  description = "The ssl policy name that certificate must follow"
```

Comment:
Maybe we can make 'SSL' uppercase here and put a 'the' in front of certificate? :)